.github/workflows/live-game-fe.yml

```
@@ -99,6 +101,8 @@
           VITE_LISTEN_SITE_URL: ${{ vars.VITE_LISTEN_SITE_URL }}
           VITE_WATCH_SITE_URL: ${{ vars.VITE_WATCH_SITE_URL }}
           VITE_PLAY_SITE_URL: ${{ vars.VITE_PLAY_SITE_URL }}
+          VITE_ROLLBAR_TOKEN: ${{ github.VITE_ROLLBAR_TOKEN }}
```

actionlint (1.7.4)

property "vite_rollbar_token" is not defined in object type {action: string; action_path: string; action_ref: string; action_repository: string; action_status: string; actor: string; actor_id: string; api_url: string; base_ref: string; env: string; event: object; event_name: string; event_path: string; graphql_url: string; head_ref: string; job: string; job_workflow_sha: string; path: string; ref: string; ref_name: string; ref_protected: string; ref_type: string; repository: string; repository_id: string; repository_owner: string; repository_owner_id: string; repositoryurl: string; retention_days: number; run_attempt: string; run_id: string; run_number: string; secret_source: string; server_url: string; sha: string; token: string; triggering_actor: string; workflow: string; workflow_ref: string; workflow_sha: string; workspace: string}

(expression)